db/structure.sql

```diff
@@ -441,7 +427,7 @@ ALTER TABLE ONLY public.offices
 -- PostgreSQL database dump complete
 --
 
-SET search_path TO "$user", public, topology, tiger;
+SET search_path TO "$user", public, topology;
```

Comment:
`tiger`? 🤔

Reply:
oh I don't know why that keeps appearing, it seems a bit random. TIGER is apparently a US geodata database that I think PostGIS bundles as some dummy data so it somehow ends up in the schema dump